tests/test_serialization.py

```diff
@@ -79,6 +81,45 @@ def test_save_load__from_filename(
         save_load_from_filename(cfg, resolve, expected, file_class)
 
 
+@dataclass
+class Dataclass:
+    number: int = 2
+    done: bool = True
+
+
+@attr.s(auto_attribs=True)
+class Attr:
+    number: int = 2
+    done: bool = True
```

Comment:
There are some test dataclasses in tests/__init__.py
Use something from there (and add an attr.s one to complete the test).